### DISCUSSION TITLE
[PSD9] Domande Slide 13

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/psd2223#217] [Esercizi (A-L)]
Domande:
- Quante porte AND servono per fare uno XOR ad 8 ingressi utilizzando una rete AND/OR a 2 livelli?
- Quanti ingressi hanno le AND?
- Quanti ingressi ha l’OR che raccoglie le uscite degli AND?

**Comment by Elia-Belli** [comment]
- Uno $XOR\ 8$ ha $2^8/2 =128$ mintermini, se lo realizziamo con una rete $AND/OR$ a 2 livelli utilizziamo $128\ AND$ e $1\ OR$
- Siccome ogni mintermine ha $8$ literals, ogni $AND$ ha $8$ ingressi
- Siccome abbiamo $128$ mintermini l' $OR$ ha $128$ ingressi
